src/quo2/foundations/typography.cljs

```diff
@@ -1,25 +1,40 @@
 (ns quo2.foundations.typography)
 
-;; Formulat for letter spacing from figma %: font-size*{figma-percentage}/100
+(defn- tracking-fn
+  "Check inter dynamic metric https://rsms.me/inter/dynmetrics/"
+  [font-size]
+  (let [a -0.0223
+        b 0.185
+        c -0.1745
+        e js/Math.E]
+    (->> font-size
+         (* c)
+         (js/Math.pow e)
+         (* b)
+         (+ a)
+         (* font-size))))
+
+(def tracking (memoize tracking-fn))
+
 (def heading-1 {:font-size      27
                 :line-height    32
-                :letter-spacing -0.567})
+                :letter-spacing (tracking 27)})
```

Comment:
Hi @yqrashawn, Thank you for the PR.
Did you tried comparing resulting output with figma? Because as I mentioned in issue, text component was ignoring small changes in `letter-spacing` value.
(For comparison, please type big line. Small letter spacing might not be visible, but it adds up to a big difference with several characters)

Reply:
<img width="1549" alt="image" src="https://user-images.githubusercontent.com/15090582/189306500-de647ccd-7f40-4319-9616-162c663ab7a3.png">

Well, the `tracking` fn generates the same `letter-spacing` compaired with https://rsms.me/inter/dynmetrics/

I do compared with figma. 
In fact, I also compared figma and screen shots from https://rsms.me/inter/dynmetrics/ 
There're minor mismatches as well.

Reply:
The old calculation below is definitely incorrect. This commit fix it, yet can't remove the mismatch completely (numbers get rounded)
 https://github.com/status-im/status-mobile/blob/f1f9fa105ced7f4471311f4e6a2deace2302f722/src/quo2/foundations/typography.cljs#L3

Reply:
>I do compared with figma.
In fact, I also compared figma and screen shots from https://rsms.me/inter/dynmetrics/
There're minor mismatches as well (numbers get rounded).

Did you mean mismatches between website and figma or screenshots and figma.
Please can you attach those comparison screenshots. (So its easier to understand)
(If possible, please also attach comparison of old-text, new-text, and figma, aligned)

Reply:
@Parveshdhull check the below link for comparison between various screenshots and Figma design
https://www.figma.com/file/VHRoCTBxP7J7wwIbt3Uoto/typography-letter-spacing-%2313960

Reply:
Hi @yqrashawn,
Thank you very much for the Figma file. But I was hoping comparison of aligned texts to see the difference between letter spacing, of pr, and develop. As shown in this Figma
https://www.figma.com/file/8zjDY1vglXCpbTfc7rEsfa/Typography?node-id=0%3A1
(don't know if you can access file, so here is a [screenshot](https://user-images.githubusercontent.com/17097240/189548576-558fc1bd-6f83-4b79-ad0b-fa4b4f5132a1.png) of comparison)

So, it looks like both formulas are giving the same output. But I guess we can merge this PR as this is the official formula. But please keep the issue open as letter spacing still needs fixing. Thank you